lib/src/emoji_picker_utils.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:convert';
+import 'dart:math';
+
+import 'package:emoji_picker_flutter/emoji_picker_flutter.dart';
+import 'package:emoji_picker_flutter/src/recent_emoji.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'emoji_lists.dart' as $emoji_list;
+
+/// EmojiPicker public method class
+/// Easy to use and expand other functions
+/// Reduce repeated encapsulation of the same function
+
+class EmojiPickerUtils {
+  /// Returns list of recently used emoji from cache
+  static Future<List<RecentEmoji>> getRecentEmojis() async {
+    final prefs = await SharedPreferences.getInstance();
+    var emojiJson = prefs.getString('recent');
+    if (emojiJson == null) {
+      return [];
+    }
+    var json = jsonDecode(emojiJson) as List<dynamic>;
+    return json.map<RecentEmoji>(RecentEmoji.fromJson).toList();
+  }
+
+  /// Returns list of all the category emojis
+  static Map<Category, Map<String, String>> get getCategoryEmoji =>
+      _categoryEmoji;
+
+  static final Map<Category, Map<String, String>> _categoryEmoji = Map.from({
+    Category.ACTIVITIES: $emoji_list.activities,
+    Category.ANIMALS: $emoji_list.animals,
+    Category.FLAGS: $emoji_list.flags,
+    Category.FOODS: $emoji_list.foods,
+    Category.OBJECTS: $emoji_list.objects,
+    Category.SMILEYS: $emoji_list.smileys,
+    Category.SYMBOLS: $emoji_list.symbols,
+    Category.TRAVEL: $emoji_list.travel
+  });
```

Comment:
There two issues here
1: Like this the Map will be created during app start and not earliest when the library is actually used, or am I wrong? If it's true then I would like to rather postpone it.

2: This list includes all emojis unfiltered, even those that aren't supported on that device. I think this would make it difficult to implement a search functionality without showing some 🖾  instead. Wouldn't it be better to provide filtered list?